src/screens/profile/home/account-delete-modal.tsx

```diff
@@ -16,37 +23,38 @@ const AccountDeleteModal: React.FC<AccountDeleteModalProps> = ({
   isModalOpen,
   setIsModalOpen,
 }) => {
-  const { height } = useWindowDimensions();
+  const theme = useTheme();
 
   const handleModalClose = () => {
     setIsModalOpen(false);
   };
 
   return (
-    <Modal _backdrop={{ height }} isOpen={isModalOpen} onClose={handleModalClose}>
-      <Modal.Content>
-        <Modal.Header>
-          <Box flexDirection={'row'} justifyContent={'space-between'}>
-            <Heading>Delete Account</Heading>
-            <Icon size={6} as={<MaterialIcons name="close" />} onPress={handleModalClose} />
-          </Box>
-        </Modal.Header>
-        <Modal.Body>
-          <Text>Are you sure you want to delete your account?</Text>
-        </Modal.Body>
-        <Modal.Footer>
-          <Button variant={'subtle'} onPress={handleModalClose}>
+    <Modal isModalOpen={isModalOpen} onRequestClose={handleModalClose}>
+      <ModalHeader title="Delete Account" onClose={handleModalClose} />
+      <ModalBody>
+        <Box alignItems="center">
+          <Text textAlign={'center'}>Are you sure you want to delete your account?</Text>
+        </Box>
+      </ModalBody>
+      <ModalFooter>
+        <Box flexDirection="row" justifyContent="space-between" alignItems="center" width="100%">
+          <Button onClick={handleModalClose} kind={ButtonKind.SECONDARY} width="48%">
             Cancel
           </Button>
           <Button
             isLoading={isDeleteAccountLoading}
-            onPress={handleDeleteAccountPress}
-            variant={'danger'}
+            onClick={handleDeleteAccountPress}
+            kind={ButtonKind.DANGER}
+            width="48%"
+            startEnhancer={
+              <DeleteIcon width={20} height={20} fill={theme.colors.secondary['50']} />
```

Comment:
I do not fully understand usage of `50` etc. I would think we have standard colors - ex, bootstrap provides these [standard colors](https://getbootstrap.com/docs/5.3/utilities/colors/#colors).

How can one figure out which color to use?